### PR TITLE
DE4135 - don't send the default email when 'trying a group'

### PR DESCRIPTION
**Pull request by plachmann**

### CODE REVIEW

File: Gateway/crds-angular/Services/GroupToolService.cs [hunk 1]
```diff
@@ -42,6 +42,7 @@ public class GroupToolService : MinistryPlatformBaseService, IGroupToolService
         private readonly int _defaultAuthorUserId;
         private readonly int _defaultGroupRoleId;
         private readonly int _groupRoleLeaderId;
+        private readonly int _groupRoleTrialMemberId;
```
Comment (vchtelmakh) [inline Gateway/crds-angular/Services/GroupToolService.cs]: Dank of jank.

File: Gateway/crds-angular/Services/GroupToolService.cs [hunk 2]
```diff
@@ -111,6 +112,7 @@ IFinderRepository finderRepository
             _defaultGroupContactEmailId = configurationWrapper.GetConfigIntValue("DefaultGroupContactEmailId");
             _defaultAuthorUserId = configurationWrapper.GetConfigIntValue("DefaultAuthorUser");
             _groupRoleLeaderId = configurationWrapper.GetConfigIntValue("GroupRoleLeader");
+            _groupRoleTrialMemberId = configurationWrapper.GetConfigIntValue("GroupsTrialMemberRoleId");
```
Comment (vchtelmakh) [inline Gateway/crds-angular/Services/GroupToolService.cs]: Janky.

Comment (plachmann) [inline Gateway/crds-angular/Services/GroupToolService.cs]: Dejankified and pushed

File: Gateway/crds-angular/Services/GroupToolService.cs (hunk 1 above)
Comment (plachmann) [inline Gateway/crds-angular/Services/GroupToolService.cs]: Dejankified and pushed

**Pull request merged**